classes/Task/Update/UpdateComplete.php

```diff
@@ -62,7 +62,7 @@ public function run(): int
 
         $this->next = TaskName::TASK_COMPLETE;
 
-        if ($this->container->getUpgradeConfiguration()->getChannel() != Upgrader::CHANNEL_LOCAL && file_exists($this->container->getFilePath()) && unlink($this->container->getFilePath())) {
+        if ($this->container->getUpgradeConfiguration()->getChannel() != UpgradeConfiguration::CHANNEL_LOCAL && file_exists($this->container->getFilePath()) && unlink($this->container->getFilePath())) {
```

Comment:
Worth using your new methods?

```suggestion
        if ($this->container->getUpgradeConfiguration()->isChannelOnline() && file_exists($this->container->getFilePath()) && unlink($this->container->getFilePath())) {
```